lib/active_model_serializers/adapter/json_api.rb

```diff
@@ -48,9 +48,9 @@ def serializable_hash(options = nil)
         document = if serializer.success?
                      success_document(options)
                    else
-                     failure_document
+                     failure_document(options)
                    end
-        transform_key_casing!(document, options[:serialization_context])
```

Comment:
When I introduced this I thought it made more sense to pass just the `serialization_context` from the options. Now I think it makes more sense to just pass the options.